src/main/java/ch/njol/skript/effects/EffPlaySound.java

```diff
@@ -240,4 +239,27 @@ public String toString(@Nullable Event event, boolean debug) {
 		return builder.toString();
 	}
 
+	@SuppressWarnings({"deprecation", "unchecked", "rawtypes"})
```

Comment:
this should be per statement. you should use the noinspection comment (alt enter > arrow down (suppress) > suppress for statement)

Reply:
I don't see the point of this. Why remove 1 line to create 3+ more lines?
And since when did SkriptLang do this? I dont think ive ever done this nor ever seen it, nor is it in the code conventions.

Reply:
i think using suppresswarnings on an entire method may lead to confusion as to when something generally considered not good is used (like casting raw types etc). using these comments just emphasizes where they actually occur

Reply:
So is this a SkriptLang preference, or personal preference?
I personally think having a suppression for a block makes more sense than adding a suppression for every line in the block.

Reply:
It's something we agreed on back in august or so. If it's not in the conventions yet it probably needs to be added. Warning suppression should be as specific as possible to  avoid suppressing other warnings that weren't originally intended (see suppressing one deprecated method, but then a second gets deprecated too, it's good to still see the warning for the second.)
Of course, if the warnings are routine and/or the block is very small, like many `init()` methods, it's not worth suppressing each line individually.
In this case I can see it either way, I don't think it matters much, since the method's quite small.

Reply:
> If it's not in the conventions yet it probably needs to be added.

It is :)

Reply:
![image](https://github.com/user-attachments/assets/94cf18ca-5dd3-4b69-8877-de5e34773743)
ah it's in the nullness category instead of the formatting category